libs/SmartStore/src/com/salesforce/androidsdk/smartstore/store/DBOpenHelper.java

```diff
@@ -261,18 +262,11 @@ public void onUpgrade(SQLiteDatabase db, int oldVersion, int newVersion) {
 		if (oldVersion == 1) {
 			SmartStore.createLongOperationsStatusTable(db);
 		}
-
```

Comment:
If we are getting rid of this block, shouldn't we also get rid of the one above (line 261) ?

Reply:
Yes - I could.